### DISCUSSION TITLE
is Using extendTheme API is safe for theming in future?

### DISCUSSION THREAD

**Discussion by roker15** [chakra-ui/chakra-ui#6769] [Q&A]
I saw this line in VS Code while using `extendTheme`

`NOTE: This got too complex to manage, and it's not worth the extra complexity. We'll re-evaluate this API in the future releases.`

1. So What is road map for this?
2. Is Chakra planning to obsolete theming and going to leave it to raw CSS file? Or Coming with alternative?
3. How much will it break existing code using `extendTheme `?

**Comment by valhovey** [comment]
Also wondering about this: we are using `extendTheme` quite a bit to align Chakra to our design system. Is there another recommended way we should be doing this?
